### PR TITLE
Label format

### PR DESCRIPTION
**Pull request by IKrukov-HORIS**

### CODE REVIEW

File: plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt [hunk 1]
```diff
@@ -38,6 +40,17 @@ class TextGeom : GeomBase() {
...
 
+    private fun toString(label: Any): String {
+        if (label is String)
+            return label
+
+        if (label is Double) {
+            return formatter?.apply(label) ?: label.toString()
+        }
+
+        throw IllegalStateException("Unacceptable label type.")
```
Comment (alshan) [inline plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt:51]: This id too much. Let's return `toString` value.

Comment (IKrukov-HORIS) [reply plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt]: Fixed.

File: plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt [hunk 2]
```diff
@@ -38,6 +40,14 @@ class TextGeom : GeomBase() {
         }
     }
 
+    private fun toString(label: Any): String {
+        if (label is Double) {
+            formatter?.let { return it.apply(label) }
+        }
+
+        return label.toString()
```
Comment (alshan) [inline plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt:48]: Let's return string "n/a" for nulls.

File: plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt [hunk 3]
```diff
@@ -38,6 +40,14 @@ class TextGeom : GeomBase() {
...
+    }
+
     companion object {
 //        val RENDERS = listOf(
```
Comment (alshan) [inline plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt:52]: Please cleanup.

Comment (IKrukov-HORIS) [reply plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt]: Done

File: plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt (hunk 2 above)
Comment (IKrukov-HORIS) [reply plot-base-portable/src/commonMain/kotlin/jetbrains/datalore/plot/base/geom/TextGeom.kt:53]: Done

**Pull request merged**